nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -150,6 +152,7 @@
     private ExtensionMapping extensionMapping;
     private NarAutoLoader narAutoLoader;
     private DiagnosticsFactory diagnosticsFactory;
+    private SslContextFactory sslContextFactory;
```

Comment:
Should the type be `SslContextFactory.Server`?
```suggestion
    private SslContextFactory.Server sslContextFactory;
```

Reply:
I think it could be either, but since SslContextFactory.Server extends SslContextFactory, I figured it would be better to have the type be the parent.